src/database/db_build.sql

```diff
@@ -6,90 +6,71 @@ DROP TABLE IF EXISTS users, Projects, coordination_status, project_status CASCAD
 CREATE TABLE users (
 	Id SERIAL UNIQUE PRIMARY KEY,
 	name varchar(100) NOT NULL,
-	username varchar(100) UNIQUE NOT NULL,
+	username varchar(100) NOT NULL,
 	pass TEXT NOT NULL,
-	Email TEXT UNIQUE NOT NULL,
-	id_number integer UNIQUE NOT NULL,
-	mobile integer UNIQUE NOT NULL,
+	Email TEXT NOT NULL,
```

Comment:
Can you change `Email` to `email`? Since you are writing all names in lowercase so you might want to make this one lowercase as well.

Same for `Id`